### DISCUSSION TITLE
Unable to view my frontend logs

### DISCUSSION THREAD

**Discussion by bharath-gaveni** [learndevopsonline/learndevopsonline#1624] [Q&A] [answered]
Unable to view my frontend logs 

![Screenshot (17)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/fbfd0161-a66d-4418-a06a-858467f31a36)
![Screenshot (16)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/36fa4cab-acf2-47e8-8e18-d9c50c479911)

**Comment by sreedharm07** [comment]
hi @bharath-gaveni 
your file beat was not able to start 
please configure your filebeat properly
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/a3fd3bfd-c44c-4de0-abc6-7cff7f96b95a)

**Comment by bharath-gaveni** [comment]
![Screenshot (19)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/71977c0d-2a08-46df-b269-b8cb63f5f902)
filebeat is started but not showing logs

**Reply by sreedharm07** [reply]
in you screenshot logs are showing.
 what logs not showing?

**Comment by bharath-gaveni** [comment]
failing to connect to backoff

**Reply by sreedharm07** [reply]
please dont use multiple threads 
use only one thread

**Comment by bharath-gaveni** [comment]
in elastic search its not showing

**Reply by sreedharm07** [reply]
give your elk ip

**Reply by sreedharm07** [reply]
and again you are using multiple thread
please use one thread

**Reply by sreedharm07** [reply]
am replying in one thread and you are again replying in one thread it will confuse you and me use single thread

**Comment by bharath-gaveni** [comment]
44.211.79.105

**Reply by sreedharm07** [reply]
again you opened new thread

**Reply by bharath-gaveni** [reply]
sorry

**Reply by sreedharm07** [reply, marked as answer]
please give your password here 
in logstash.conf
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/b3c64d08-da37-4b1e-8036-6dc1d3d467eb)
and 
read all this before you creating discussion.
becasue you are creating multiple threads
https://github.com/learndevopsonline/learndevopsonline/discussions/new?category=q-a

**Reply by bharath-gaveni** [reply]
I am just confused

**Reply by bharath-gaveni** [reply]
okay sure thank you for you patience @sreedharm07
